Makefile

```diff
@@ -78,8 +79,8 @@ register: _requires-sandbox-up  ## Register Flyte cookbook workflows
 
 .PHONY: fast_register
 fast_register: _requires-sandbox-up  ## Fast register Flyte cookbook workflows
-	$(call LOG,Fast registering example workflows in cookbook/$(EXAMPLES_MODULE))
-	$(call RUN_IN_SANDBOX,make -C cookbook/$(EXAMPLES_MODULE) fast_register)
+	$(call LOG,Fast registering example workflows from latest release of flytesnacks)
```

Comment:
What if someone has updated the examples locally and wants to fast register those, why always on latest release

Reply:
There is a command that exists in the core example that can be used. This target only cover the core example